LayoutTests/fast/text/text-selection-direction-overflow-crash.html

```diff
@@ -0,0 +1,15 @@
+<style>
+.classA { rotate: 60deg; min-height: 38%; font-size: 1px; display: inline-flex; -webkit-transform: translate(0px, 0px) scale(83); clip-path: polygon(0px 0px, 1px 0px); scale: 0.29403306255747363 32 7; -webkit-transform-origin: top left; }
```

Comment:
This is the reduced version of the test case attached to the radar - reducing it any further doesn't cause the crash. Let me know if you folks have any other ideas for reducing it.

Reply:
I think landing this is fine, but I also suspect it might be able to be reduced further.  I'm not sure whether the time required to do reduce it more would be worth it in this case.

Reply:
Yeah, thanks, removing even a single style from this gets rid of the issue, but maybe @nt1m has some thoughts?

Reply:
I usually try to resolve relative values and remove potential duplicates (scale?), but I think it is fine as is.

Reply:
I would remove the -webkit- prefixes, they're unnecessary, along with finding more simple values as Alan says.